src/plugins/tracking/_activityTrackers.js

```diff
@@ -95,36 +89,31 @@ export const getTrackApiRequest: TrackGetter = (trackOpts: TrackOpts) => (
  * get the trackApi handler - the core tracking handler for navigation and
  * login-related activity.
  */
-export const getTrackApi: TrackGetter = (trackOpts: TrackOpts) => {
-	const trackApiRequest = getTrackApiRequest(trackOpts);
-	const trackLogin = getTrackLogin(trackOpts);
-	const trackLogout = getTrackLogout(trackOpts);
-
-	return (request: Object, queryResponses: Array<Object>) => {
-		const payload = request.method === 'post' ? request.payload : request.query;
+export const getTrackApi: TrackGetter = trackOpts => request => (
+	queryResponses: Array<Object>
+) => {
+	const payload = request.method === 'post' ? request.payload : request.query;
 
-		const metadataRison = payload.metadata || rison.encode_object({});
```

Comment:
this is all an indentation change because we no longer need to create local `track{ApiRequest|Login|Logout}` functions - they are now `request` methods